tds/field.go

```diff
@@ -86,6 +86,9 @@ type FieldFmt interface {
 	// Format information as sent to or received from TDS server
 	DataType() asetypes.DataType
 	setDataType(asetypes.DataType)
+	// ConvertDataType converts a datatype to the passed nullable
+	// datatype and sets according status and length
+	ConvertDataType(asetypes.DataType)
```

Comment:
Does the passed datatype have to be nullable? Looking at the code at a glance it doesn't seem like it.

Reply:
Change to `ConvertToNullableDataType`